BEP-20.md

```diff
@@ -0,0 +1,84 @@
+
+3. **Faster Vetting** Vetting time for investors, market liquidity and orderbook integrity are interdependent. Some vetting processes for private securities have historically taken months - delaying transactions significantly. By pre-vetting investors and increasing the number of matched orders that will settle without complication, security token markets can become more liquidly tradeable than their historical private market counterparts. Whitelists enable these conditions.
+
+## Whitelist Limitations
+
+The issuer controlled whitelist proposed here only directly addresses the digital asset issuer's regulatory burden for restricting token transfers. In addition to the issuer's responsibility, each exchange will need to confirm the AML/KYC and accreditation status of each trader in keeping with regulations that apply to them. Exchanges and marketplaces are directly accountable for their own vettings. What is gained by the whitelist is pre-approval by the issuer for compliant trades.
+
+Whitelists are only suitable for unrestricted trading between accredited investors that do not violate other restrictions like maximum holders, maximum purchase amounts or trading restrictions between specific jurisdictions. Fine grained control of transfer restricitons will need to be addressed in a future transfer restrictions BEP. These restrictions are especially relevant for unacreddited ("everyday") investors.
+
+## 5.  Specification
+
+### 5.1 Token Attributes For Whitelist
+
+* Tokens have a `whitelist_required_for_transfer` attribute of type `boolean`
+* Tokens have a `whitelist` of type `[address]`
+
+### 5.2 Token Whitelist Attribute Access Restrictions
+* Only the token owner can set the `whitelist_required_for_transfer` boolean value
```

Comment:
A token owner might lose his private key. If this happens, the whitelist can be changed any more.  Do you think it is necessary to enable a token owner to specify more manage addresses which can change the whitelist too?

Reply:
Yes, great point. This would be a serious problem. I think the solution for this is native mult-signature support for binance chain private keys. Maybe this goes in a new BEP? 

In the mean time token owners would need to find a native threshold signature or key sharding solution for secp256k1 keys. If there is a multi-sig wallet solution like this available for Binance Chain then that may be the initial recommended best practice.